docs/date.md

```diff
@@ -1,96 +1,398 @@
 # Temporal.Date
 
-A representation of a calendar date.
+A `Temporal.Date` represents a calendar date.
+"Calendar date" refers to the concept of a date as expressed in everyday usage, independent of any time zone.
+For example, it could be used to represent an event on a calendar which happens during the whole day no matter which time zone it's happening in.
+
+`Temporal.Date` refers to the whole of a specific day; if you need to refer to a specific time on that day, use `Temporal.DateTime`.
+A `Temporal.Date` can be converted into a `Temporal.DateTime` by combining it with a `Temporal.Time` using the `withTime()` method.
+
+`Temporal.YearMonth` and `Temporal.MonthDay` carry less information than `Temporal.Date` and should be used when complete information is not required.
 
 ## Constructor
 
 ### **new Temporal.Date**(_year_: number, _month_: number, _day_: number, _disambiguation_: 'constrain' | 'balance' | 'reject' = 'constrain') : Temporal.Date
 
-Creates a new `Date` object that represents a calendar date
+**Parameters:**
+- `year` (number): A year, ranging between &minus;271821 and 275760 inclusive.
```

Comment:
Where do these numbers come from?

Reply:
I'd be interested to know this too

Reply:
It comes from legacy Date, cf. https://tc39.es/ecma262/#sec-time-values-and-time-range
> A Number can exactly represent all integers from -9,007,199,254,740,992 to 9,007,199,254,740,992 (<a href="https://tc39.es/ecma262/#sec-number.min_safe_integer">20.1.2.8</a> and <a href="https://tc39.es/ecma262/#sec-number.max_safe_integer">20.1.2.6</a>). A time value supports a slightly smaller range of -8,640,000,000,000,000 to 8,640,000,000,000,000 milliseconds. This yields a supported time value range of exactly -100,000,000 days to 100,000,000 days relative to midnight at the beginning of 01 January, 1970 UTC.

An explanatory reference would be useful.

Reply:
This came from #24. However according to my own principle below I should've left this out — currently the limit is -999999 to +999999 in the polyfill and spec. I'll remove this for this revision and it can be added to the docs as part of #24.